vamb/__main__.py

```diff
@@ -1275,6 +1649,55 @@ def main():
         help="binsplit separator [None = no split]",
     )
 
+    # Taxonomy arguments
+    taxonomys = parser.add_argument_group(
+        title="Taxonomy (results of mmseqs search)"
+    )
+    taxonomys.add_argument(
+        "--taxonomy", 
+        metavar="", 
+        type=Path, 
+        help="path to taxonomy tsv file, output of mmseq search",
+    )
+    taxonomys.add_argument(
+        "--taxonomy_predictions", 
+        metavar="", 
+        type=Path, 
+        help="path to results_taxonomy_predictor.csv file, output of taxonomy_predictor model",
+    )
+    taxonomys.add_argument(
+        "--no_predictor", 
+        metavar="", 
+        type=bool, 
+        default=False,
+        help="do not complete mmseqs search with taxonomy predictions",
+    )
+
+    # Reclustering arguments
+    reclusters = parser.add_argument_group(
+        title="k-means reclustering arguments"
+    )
+    reclusters.add_argument(
+        "--latent_path", 
+        metavar="", 
+        type=Path, 
+        help="path to a latent space",
+    )
+    reclusters.add_argument(
+        "--clusters_path", 
+        metavar="", 
+        type=Path, 
+        help="path to a cluster file corresponding to the latent space",
+    )
+    reclusters.add_argument(
+        "-ro",
+        dest="binsplit_separator_recluster",
+        metavar="",
+        type=str,
+        default='C',
+        help="binsplit separator for reclustering [C]",
+    )
```

Comment:
I wonder if this needs to be able to be different than the normal binsplit separator.

Reply:
I made a decision of moving it here because I didn't like how the parameters `clustering_options` and `reclustering_options` look next to each other :) but it should just be a part of `clustering_options`, especially if those are reused from a different model

The `--reclustering` flag we can also use with AVAMB models. VAMB showed a big improvement from reclustering for the short read, a smaller but solid for the long reads